domain-model.md

```diff
@@ -0,0 +1,55 @@
+Domain Model - Cohort Manager 
+
+The Cohort Manager should be able to support the following interactions
+
+- Create a cohort with a cohort name
+- Search for a cohort by cohort name
+- Add student to a specific cohort
+- Remove a cohort by cohort name
+- Remove student from a specific cohort
+- Return errors if student or cohort not found
+
+A cohort should have a list of students. Each student should have a studentID, first name, last name, github username, email.
+```
+
```

Comment:
Well thought out domain model, writing a comment describing what function will do is looks really helpful